Makefile

```diff
@@ -0,0 +1,57 @@
+.PHONY: build lint test clean run_image build_image download-vectors clean-vectors \
+	setup-hive test-pattern-default run-hive run-hive-debug clean-hive-logs
+
+build:
+	cargo build --workspace
+
+lint:
+	cargo clippy --all-targets --all-features --workspace -- -D warnings
+
+CRATE ?= *
+test:
+	cargo test -p '$(CRATE)'
+
+clean:  clean-vectors
+	cargo clean
+	rm -rf hive
+
+run_image: build_image
+	docker run --rm -p 127.0.0.1:8545:8545 ethereum_rust --http.addr 0.0.0.0
+
+build_image:
+	docker build -t ethereum_rust .
+
+SPECTEST_VERSION := v3.0.0
+SPECTEST_ARTIFACT := tests_$(SPECTEST_VERSION).tar.gz
+SPECTEST_VECTORS_DIR := cmd/ef_tests/vectors
+
+$(SPECTEST_VECTORS_DIR):
+	curl -L -o $(SPECTEST_ARTIFACT) "https://github.com/ethereum/execution-spec-tests/releases/download/$(SPECTEST_VERSION)/fixtures_stable.tar.gz"
+	mkdir -p $(SPECTEST_VECTORS_DIR) tmp
+	tar -xzf $(SPECTEST_ARTIFACT) -C tmp
+	mv tmp/fixtures/blockchain_tests/* $(SPECTEST_VECTORS_DIR)
+	rm -rf tmp $(SPECTEST_ARTIFACT)
+
+download-vectors: $(SPECTEST_VECTORS_DIR)
```

Comment:
we should add SPECTEST_VERSION as a dependency so that `download-vectors` is re-executed if the version changes

Reply:
Making SPECTEST_ARTIFACT a dependency works for me!